convert2rhel/toolopts.py

```diff
@@ -65,15 +66,20 @@ def _get_argparser():
                  "  convert2rhel [-k key] [-o organization] [--pool pool_id |"
                  " -a] [--disablerepo repoid] [--enablerepo repoid] [-v"
                  " variant] [--serverurl url] [--no-rpm-va] [--debug]"
-                 " [--restart] [--disable-colors] [-y]")
+                 " [--restart] [--disable-colors] [-y]"
+                 "\n\n"
+                 "WARNING: The tool needs to be run under the root user"
+                 )
         return optparse.OptionParser(conflict_handler='resolve',
                                      usage=usage,
-                                     add_help_option=False)
+                                     add_help_option=False,
+                                     version=__version__)
 
     def _register_options(self):
         """Prescribe what command line options the tool accepts."""
         self._parser.add_option("-h", "--help", action="help", help="Show "
                                 " help message and exit.")
+        self._parser.add_option('--version', action='version', help="Show version of the application.")
```

Comment:
```suggestion
        self._parser.add_option('--version', action='version', help="Show Convert2RHEL version and exit.")
```

Reply:
In this case I'd maybe use the lower case "convert2rhel" to emphasize we talk about the cli utility and not the project.

Reply:
Fixed

Reply:
also fixed